src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  @include flex(center, space-between);
```

Comment:
The use of `@include flex(center, space-between);` suggests that a mixin named `flex` is being used. Ensure that this mixin is defined in your project, as it is not a standard CSS property.